catalog/transformer/catalog-transformer-html/src/main/java/ddf/catalog/transformer/html/models/HtmlExportCategory.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.html.models;
+
+import ddf.catalog.data.Metacard;
+import java.util.List;
+import java.util.Map;
+
+/**
```

Comment:
We may want to mark this interface as experimental
https://github.com/codice/ddf/blob/cec05201f98ba223098d93ea9203b4cbe8b4deef/catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/Suggestion.java#L16